README.md

```diff
@@ -48,3 +48,21 @@ For more information about Compose CLI options, see Docker Compose documentation
 ### `irods-client-zmt` / Zone Management Tool / ZMT
 
 The ZMT service assumes that containers are running on the same host as the browser. If this is not the case, the value of `REACT_APP_REST_API_URL` should be modified to an address which correctly maps to the `nginx-reverse-proxy` service and that is reachable by both the ZMT service and the host running the browser.
+
+### `irods-client-nfsrods` / NFSRODS
+
+Once the service is running, the NFS server needs to be accessed from a mountpoint on the host machine. This can be done with the following command:
+```bash
+$ sudo mount -o sec=sys,port=2049 localhost:/ ./irods_client_nfsrods/nfs_mount
+```
+The hostname can also be the IP address of the container providing the service in the `irods_demo_default` Docker network. For more information about mounting NFSRODS, see the README for the project: https://github.com/irods/irods_client_nfsrods#mounting
```

Comment:
A third option... a separate machine elsewhere can mount NFSRODS by using the host machine's FQDN or IP as well?   Since port 2049 is exposed?

Reply:
True. Will update wording

Reply:
or 2050 now